app/src/main/java/com/example/game/fragments/SignupFragment.java

```diff
@@ -77,6 +82,8 @@ public void onClick(View view) {
                         public void done(ParseException e) {
                             if (e != null) {
                                 Log.e(TAG, "Error Signup: " + e);
+                                progressBar.setVisibility(View.INVISIBLE);
```

Comment:
Is this on purpose? invisible will still take certain part of the the host view

Reply:
yes, it is part of the design. I still want that space.